### PR TITLE
XIVY-9123 UI Revision N&N chapter

### PR DESCRIPTION
**Pull request by ivy-lli**

### CODE REVIEW

File: src/app/pages/news/9.4/03-ui-revisions.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+## UI Revisions {#ui-revision}
+
+We refurbished our general look and feel with a new PrimeFaces version, a new
+default Theme and a complete new way to brand an Axon Ivy Engine according to
```
Comment (phochstr) [inline src/app/pages/news/9.4/03-ui-revisions.md]: according --> for

File: src/app/pages/news/9.4/03-ui-revisions.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+your company.
+
+- __PrimeFaces 11__: We updated from PrimeFaces 7 to 11, which means a lot of
+  new and updated widgets. Be aware of our separate [PrimeFaces
+  Migration-Guide](${docBaseUrl}/axonivy/migration/migration-notes-94.html#primefaces-11).
+- __Freya Theme__: We also provide the [Freya
+  Theme](https://www.primefaces.org/layouts/freya) as new default theme for your
+  HTML Dialogs. The refreshed look of it also supports dark mode and of course
```
Comment (phochstr) [inline src/app/pages/news/9.4/03-ui-revisions.md]: The refreshed look of it --> Its fresh look

File: src/app/pages/news/9.4/03-ui-revisions.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  we updated the `Dev-Workflow-UI` and the `Engine Cockpit` also to this new
```
Comment (phochstr) [inline src/app/pages/news/9.4/03-ui-revisions.md]: also to this --> also to use this

File: src/app/pages/news/9.4/03-ui-revisions.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  theme.
+- __Branding__: There is a complete new way to
```
Comment (phochstr) [inline src/app/pages/news/9.4/03-ui-revisions.md]: complete --> completely

File: src/app/pages/news/9.4/03-ui-revisions.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+  [brand](${docBaseUrl}/designer-guide/user-interface/branding/index.html) an Axon
+  Ivy Engine according to your company colors and logo, without the need to
```
Comment (phochstr) [inline src/app/pages/news/9.4/03-ui-revisions.md]: according to your company colors and logo --> with the colors and logo of your company

**Review by phochstr — APPROVED** [review]
I changed the file and found a second complete --> completely that I changed without comment.

Ready to merge.

**Comment by ivy-lli** [comment]
Thanks @phochstr :+1:

**Pull request merged**